### ISSUE TITLE
[Russian Ruble] symbol after number

### ISSUE THREAD

**Issue by Disinterpreter** [leekchan/accounting#18]
https://github.com/leekchan/accounting/blob/17a4ce5f94d4d6b8115c2c6f39f7f2e8cb8f27f0/locale.go#L152

subj

References: 
https://core.telegram.org/bots/payments#supported-currencies
![image](https://user-images.githubusercontent.com/3816155/105159172-d4f6f400-5b1f-11eb-92b1-8fd358a822d9.png)
![image](https://user-images.githubusercontent.com/3816155/105159316-0079de80-5b20-11eb-9879-37dae586dc8a.png)
![image](https://user-images.githubusercontent.com/3816155/105159385-14254500-5b20-11eb-8026-c308f7aa30fc.png)

**Comment by leekchan** [comment]
Hello. 
Thank you for creating the issue. I don't have any knowledge of the Russian ruble, so could you explain the details of the issue in more detail? (It is always welcome to make a PR also 😄)

Thank you.

**Comment by Disinterpreter** [comment]
Ok, sorry. I just linked a string in a go code. A symbol of russian ruble is putting **after** a number of price. But in this code it is before a number.

> 
> 
> Hello.
> Thank you for creating the issue. I don't have any knowledge of the Russian ruble, so could you explain the details of the issue in more detail? (It is always welcome to make a PR also 😄)
> 
> Thank you.

**Comment by Disinterpreter** [comment]
Same for EUR. Look at this for example:
https://core.telegram.org/bots/payments#supported-currencies

**Comment by vlcty** [comment]
Yes I also stumbled over this. You can simply set your own format string according to the README. Here is my "workaround":

```go
money_formatter_2 := accounting.Accounting{Symbol: "€", Precision: 2}
money_formatter_2.SetFormat("%v %s")

fmt.Println(ac.FormatMoney(13.37))
// -> 13,37 €
```

**Comment by vlcty** [comment]
You can set your own format string according to the README. Here is my "workaround" for €:

```go
money_formatter_2 := accounting.Accounting{Symbol: "€", Precision: 2, Thousand: ".", Decimal: ","}
money_formatter_2.SetFormat("%v %s")
```

**Comment by leekchan** [comment]
Sorry for the late reply. Yes. For now, as suggested by @vlcty, it would be better to do formatting directly with `SetFormat`. I've understood we need to improve the `LocaleInfo`. Thank you.
